src/nxt_http_return.c

```diff
@@ -72,13 +77,21 @@ nxt_http_action_t *
 nxt_http_return(nxt_task_t *task, nxt_http_request_t *r,
     nxt_http_action_t *action)
 {
-    nxt_http_field_t        *field;
+    nxt_int_t               ret;
+    nxt_str_t               loc;
+    nxt_http_return_ctx_t   *ctx;
     nxt_http_return_conf_t  *conf;
 
     conf = action->u.conf;
 
-    nxt_debug(task, "http return: %d (loc: \"%V\")",
-              conf->status, &conf->location);
+    if (conf->location != NULL) {
+        nxt_var_raw(conf->location, &loc);
+
+    } else {
+        loc = (nxt_str_t) {0};
```

Comment:
@hongzhidao, this fixes the uninitialized problem mentioned in the other thread.  Do you like the initialization this way?

Reply:
Take this as example, I prefer to use `nxt_str_null()` instead.
```
     if (conf->types == NULL) {
            nxt_str_null(&exten);

        } else {
            nxt_http_static_extract_extension(shr, &exten);
            mtype = nxt_htt
```